src/components/Icon.tsx

```diff
@@ -69,6 +69,7 @@ export const IconAdvance = (props: IconAppearanceProps) => <Icon name="advance"
 export const IconAnalytics = (props: IconAppearanceProps) => <Icon name="analytics" {...props} />
 export const IconBadge = (props: IconAppearanceProps) => <Icon name="badge" {...props} />
 export const IconCheck = (props: IconAppearanceProps) => <Icon name="check" {...props} />
+export const IconCircleXmark = (props: IconAppearanceProps) => <Icon name="circle-xmark" {...props} />
```

Comment:
IconDismiss?

Reply:
Ik vind het zelf niet zo geslaagd dat we eigen namen verzinnen voor icons. In het design hebben icons ook geen functionele namen. Ik stel dus voor om hiermee op te houden, en gewoon de font awesome naam te gebruiken. Dat maakt het ook een stuk gemakkelijker om te checken of we een bepaald icon al hebben.

Btw, viel me nog een doublure op:  we gebruiken `faCheckCircle` en `faCircleCheck`, maar dat is hetzelfde icon. Zouden we wat sneller detecteren als we `circle-check` als icon-naam gebruiken, ipv `success` en `validation-accepted-by-admin`.

Ik was hier al begonnen met de nieuwe benaming, maar misschien moeten we het wat en hoe even afstemmen.

Reply:
Ja, ik merk ook dat er veel moeite gestopt in het abstract maken van die termen, maar we krijgen er weinig moois voor terug. Dus wat mij betreft gebruiken we idd gewoon fontawesome jargon. Komt er dan nog een grote PR die alles gaat hernoemen?

Reply:
Okay, dan laat ik de namen hier staan. 

Zo'n grote PR is wel goed om een keer te doen. Vervelend karweitje, in 3 repo's dus veel hulp bij refactoring is er niet, maar het maakt het leven wel wat gemakkelijker.